### PR TITLE
DTSPO-4182 RD AAT Flux v2 Image Automation

### PR DESCRIPTION
**Pull request by chrisorisawayi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-4182

### Change description ###
Added Flux v2 image automation for RD AAT.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: k8s/namespaces/rd/rd-caseworker-ref-api-integration/rd-caseworker-ref-api-integration.yaml [hunk 1]
```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/caseworker-ref-api:prod-ea5bf8f-20210824110521
+      image: hmctspublic.azurecr.io/rd/caseworker-ref-api:prod-ea5bf8f-20210824110521   #{"$imagepolicy": "flux-system:rd-caseworker-ref-api-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-caseworker-ref-api-integration/rd-caseworker-ref-api-integration.yaml:15]: Can you change this policy to `rd-caseworker-ref-api`? It looks to be using the same, I would remove the image repo/policy for `rd-caseworker-ref-api-integration`

File: k8s/namespaces/rd/rd-judicial-api-integration/rd-judicial-api-integration.yaml [hunk 2]
```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/judicial-api:prod-5f536da-20210827120457
+      image: hmctspublic.azurecr.io/rd/judicial-api:prod-5f536da-20210827120457   #{"$imagepolicy": "flux-system:rd-judicial-api-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-judicial-api-integration/rd-judicial-api-integration.yaml:15]: Can you change this policy to `rd-judicial-ref-api`? It looks to be using the same, I would remove the image repo/policy for `rd-judicial-ref-api-integration`

File: k8s/namespaces/rd/rd-judicial-data-load-integration/rd-judicial-data-load-integration.yaml [hunk 3]
```diff
@@ -14,4 +14,4 @@ spec:
   values:
     job:
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/judicial-data-load:prod-614cfd7-20210831113123
+      image: hmctspublic.azurecr.io/rd/judicial-data-load:prod-614cfd7-20210831113123   #{"$imagepolicy": "flux-system:rd-judicial-data-load-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-judicial-data-load-integration/rd-judicial-data-load-integration.yaml:17]: Can you change this policy to `rd-judicial-data-load`? It looks to be using the same, I would remove the image repo/policy for `rd-judicial-data-load-integration`

File: k8s/namespaces/rd/rd-location-ref-api-integration/rd-location-ref-api-integration.yaml [hunk 4]
```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/location-ref-api:prod-7a3ef4a-20210825101927
+      image: hmctspublic.azurecr.io/rd/location-ref-api:prod-7a3ef4a-20210825101927   #{"$imagepolicy": "flux-system:rd-location-ref-api-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-location-ref-api-integration/rd-location-ref-api-integration.yaml:15]: Can you change this policy to `rd-location-ref-api`? It looks to be using the same, I would remove the image repo/policy for `rd-location-ref-api-integration`

File: k8s/namespaces/rd/rd-location-ref-data-load-integration/rd-location-ref-data-load-integration.yaml [hunk 5]
```diff
@@ -14,4 +14,4 @@ spec:
   values:
     job:
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/location-ref-data-load:prod-359a54a-20210831113321
+      image: hmctspublic.azurecr.io/rd/location-ref-data-load:prod-359a54a-20210831113321   #{"$imagepolicy": "flux-system:rd-location-ref-data-load-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-location-ref-data-load-integration/rd-location-ref-data-load-integration.yaml:17]: Can you change this policy to `rd-location-ref-data-load`? It looks to be using the same, I would remove the image repo/policy for `rd-location-ref-data-load`

File: k8s/namespaces/rd/rd-professional-api-integration/rd-professional-api-integration.yaml [hunk 6]
```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/professional-api:prod-023945d-20210824144722
+      image: hmctspublic.azurecr.io/rd/professional-api:prod-023945d-20210824144722   #{"$imagepolicy": "flux-system:rd-professional-api-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-professional-api-integration/rd-professional-api-integration.yaml:15]: Can you change this policy to rd-location-ref-data-load? It looks to be using the same, I would remove the image repo/policy for rd-professional-api-integration

File: k8s/namespaces/rd/rd-profile-sync-integration/rd-profile-sync-integration.yaml [hunk 7]
```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/profile-sync:prod-fc75985-20210823143656
+      image: hmctspublic.azurecr.io/rd/profile-sync:prod-fc75985-20210823143656   #{"$imagepolicy": "flux-system:rd-profile-sync-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-profile-sync-integration/rd-profile-sync-integration.yaml:15]: Can you change this policy to `flux-system:rd-profile-sync`? It looks to be using the same, I would remove the image repo/policy for `flux-system:rd-profile-sync-integration`

File: k8s/namespaces/rd/rd-user-profile-api-integration/rd-user-profile-api-integration.yaml [hunk 8]
```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/user-profile-api:prod-ba55cec-20210831142043
+      image: hmctspublic.azurecr.io/rd/user-profile-api:prod-ba55cec-20210831142043   #{"$imagepolicy": "flux-system:rd-user-profile-api-integration"}
```
Comment (thomast1906) [inline k8s/namespaces/rd/rd-user-profile-api-integration/rd-user-profile-api-integration.yaml:15]: Can you change this policy to rd-user-profile-api? It looks to be using the same, I would remove the image repo/policy for `rd-user-profile-api-integration`

File: k8s/namespaces/rd/rd-caseworker-ref-api-integration/rd-caseworker-ref-api-integration.yaml (hunk 1 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-caseworker-ref-api-integration/rd-caseworker-ref-api-integration.yaml]: Yes the integration applications are using the same image.  Image policy/repo now updated.

File: k8s/namespaces/rd/rd-judicial-data-load-integration/rd-judicial-data-load-integration.yaml (hunk 3 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-judicial-data-load-integration/rd-judicial-data-load-integration.yaml]: Image policy/repo now updated.

File: k8s/namespaces/rd/rd-location-ref-api-integration/rd-location-ref-api-integration.yaml (hunk 4 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-location-ref-api-integration/rd-location-ref-api-integration.yaml]: Image policy/repo now updated.

File: k8s/namespaces/rd/rd-location-ref-data-load-integration/rd-location-ref-data-load-integration.yaml (hunk 5 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-location-ref-data-load-integration/rd-location-ref-data-load-integration.yaml]: Image policy/repo now updated.

File: k8s/namespaces/rd/rd-professional-api-integration/rd-professional-api-integration.yaml (hunk 6 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-professional-api-integration/rd-professional-api-integration.yaml]: Image policy/repo now updated.

File: k8s/namespaces/rd/rd-profile-sync-integration/rd-profile-sync-integration.yaml (hunk 7 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-profile-sync-integration/rd-profile-sync-integration.yaml]: Image policy/repo now updated.

File: k8s/namespaces/rd/rd-user-profile-api-integration/rd-user-profile-api-integration.yaml (hunk 8 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-user-profile-api-integration/rd-user-profile-api-integration.yaml]: Image policy/repo now updated.

File: k8s/namespaces/rd/rd-judicial-api-integration/rd-judicial-api-integration.yaml (hunk 2 above)
Comment (chrisorisawayi) [reply k8s/namespaces/rd/rd-judicial-api-integration/rd-judicial-api-integration.yaml]: Image policy/repo now updated.

**Review by thomast1906 — APPROVED** [review]

**Review by kavithagade12 — APPROVED** [review]

**Review by kavithagade12 — APPROVED** [review]

**Pull request merged**